### ISSUE TITLE
AlmaLinux is not yet supported

### ISSUE THREAD

**Issue by badfiles** [wazuh/wazuh-puppet#442]
On AlmaLinux hosts I get
```
Evaluation Error: Error while evaluating a Function Call, This ossec module has not been tested on your distribution (file: /etc/puppetlabs/code/environments/production/modules/wazuh/manifests/params_agent.pp, line: 432, column: 24)
```

**Comment by fcaffieri** [comment]
Hi @badfiles,
What version of Almalinux do you use and what version of wazuh have you tried to deploy, thank you for informing us of this and we will study if it is possible to support this system.

Thank you.

**Comment by badfiles** [comment]
AlmaLinux 8.5
Wazuh-agent 4.2.5-1, the problem is in puppet module, ansible module after a simple fix works.

**Comment by vcerenu** [comment]
Hello @badfiles 

The error that appears is due to the fact that within the manifest params_agent.pp we verify the versions of linux that we are supporting. Almalinux is not included in these versions, so it will not be added to the official release.
https://documentation.wazuh.com/current/upgrade-guide/compatibility-matrix/index.html#wazuh-agent

In case you have tested that wazuh-agent installs correctly in this version and you want to add it to your puppet deployment, inside the [params_agent.pp](https://github.com/wazuh/wazuh-puppet/blob/032b9e46ba45c282bf4c91d6d14d4b9467debe3d/manifests/params_agent.pp#L384) file, on line 384, the operating system family you are using is checked, you can add it there depending on what similarities it has with other OS of the same family. This manifest is located in the path /etc/puppetlabs/code/environments/production/modules/wazuh/manifests where the other Wazuh Puppet module manifests are also installed.

**Comment by alberpilot** [comment]
We will try to include AlmaLinux in the supported list as soon as possible. 

Regards, 
Alberto R

**Comment by alberpilot** [comment]
hello @badfiles 
This request was merged into the `4.3` branch. When 4.3.0 is released, AlmaLinux will be supported. 

Regards, 
Alberto R
